pkg/reconciler/configmappropagation/configmappropagation.go

```diff
@@ -341,3 +342,14 @@ func (r *Reconciler) contains(name string, list []*corev1.ConfigMap) (*corev1.Co
 	}
 	return nil, false
 }
+
+// removeOwnerReference removes the target ownerReference and returns a new slice of ownerReferences.
+func (r *Reconciler) removeOwnerReference(ownerReferences []metav1.OwnerReference, uid types.UID) []metav1.OwnerReference {
```

Comment:
No need for this to be a method. Since it doesn't use `r` it can just be a package-level function.

Reply:
done